oci/oci.go

```diff
@@ -42,6 +43,20 @@ func New(runtimePath string, containerDir string, conmonPath string, conmonEnv [
 		conmonPath:   conmonPath,
 		conmonEnv:    conmonEnv,
 	}
+
+	var err error
+	streamServerConfig := streaming.DefaultConfig
+	streamServerConfig.Addr = "0.0.0.0:10101"
+	r.streamServer, err = streaming.NewServer(streamServerConfig, r)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create streaming server")
+	}
+
+	// TODO: Is it should be started somewhere else?
+	go func() {
+		r.streamServer.Start(true)
+	}()
```

Comment:
We should make sure this goroutine has been really started before moving along. Like:
```go
ch := make(chan struct{}) 
go func() {
    close(ch)
    r.streamServer.Start(true)
}() 
<-ch
...
```